tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -212,28 +212,18 @@ protected virtual IAgentWriter GetAgentWriter(ImmutableTracerSettings settings,
         protected virtual IDiscoveryService GetDiscoveryService(ImmutableTracerSettings settings)
             => DiscoveryService.Create(settings.Exporter);
 
-        private static IDogStatsd CreateDogStatsdClient(ImmutableTracerSettings settings, string serviceName)
+        internal static IDogStatsd CreateDogStatsdClient(ImmutableTracerSettings settings, List<string> constantTags, string prefix = null)
         {
             try
             {
-                var constantTags = new List<string>
-                                   {
-                                       "lang:.NET",
-                                       $"lang_interpreter:{FrameworkDescription.Instance.Name}",
-                                       $"lang_version:{FrameworkDescription.Instance.ProductVersion}",
-                                       $"tracer_version:{TracerConstants.AssemblyVersion}",
```

Comment:
Are you sure you don't want these constant tags in your metrics? 🤔  Seems like they could be very useful for debugging if nothing else?

Reply:
@OmerRaviv